src/DacpacTool/Program.cs

```diff
@@ -210,6 +212,8 @@ private static int BuildDacpac(BuildOptions options)
                 analyzer.Analyze(packageBuilder.Model, options.Output, options.CodeAnalysisAssemblies ?? Array.Empty<FileInfo>());
             }
 
+            versionChecker.CheckForPackageUpdateAsync().GetAwaiter().GetResult();
```

Comment:
We are doing this as the last step in the process, which makes sense from a performance perspective, but if an exception occurs somewhere before this you don't get this warning, while upgrading to the latest version has the potential of resolving the exception so do we might want to do this at the start instead?